verification/curator-service/ui/src/components/landing-page/ChangePasswordForm.tsx

```diff
@@ -0,0 +1,231 @@
+import React, { useEffect, useState } from 'react';
+import { useFormik } from 'formik';
+import * as Yup from 'yup';
+
+import { makeStyles, Theme } from '@material-ui/core/styles';
+import FormHelperText from '@material-ui/core/FormHelperText';
+import FormControl from '@material-ui/core/FormControl';
+import InputLabel from '@material-ui/core/InputLabel';
+import OutlinedInput from '@material-ui/core/OutlinedInput';
+import InputAdornment from '@material-ui/core/InputAdornment';
+import IconButton from '@material-ui/core/IconButton';
+import Visibility from '@material-ui/icons/Visibility';
+import VisibilityOff from '@material-ui/icons/VisibilityOff';
+import Button from '@material-ui/core/Button';
+import Typography from '@material-ui/core/Typography';
+
+const useStyles = makeStyles((theme: Theme) => ({
+    checkboxRoot: {
+        display: 'block',
+    },
+    required: {
+        color: theme.palette.error.main,
+    },
+    inpputField: {
+        display: 'block',
+        width: '240px',
+        marginBottom: '10px',
+    },
+    signInButton: {
+        marginTop: '10px',
+    },
+    checkboxLabel: {
+        fontSize: '14px',
+    },
+    link: {
+        fontWeight: 'bold',
+        color: theme.palette.primary.main,
+        cursor: 'pointer',
+    },
+    forgotPassword: {
+        fontWeight: 'normal',
+        color: theme.palette.primary.main,
+        cursor: 'pointer',
+        fontSize: 'small',
+        marginTop: '-8px',
+        display: 'flex',
+        justifyContent: 'flex-end',
+    },
+    labelRequired: {
+        color: theme.palette.error.main,
+    },
+    title: {
+        margin: '10px 0',
+    },
+    googleButton: {
+        // margin: '35px 0 0 0',
+        fontWeight: 400,
+    },
+    formFlexContainer: {
+        display: 'flex',
+        gap: '80px',
+    },
+}));
+
+interface FormValues {
+    password: string;
+    passwordConfirmation: string;
+}
+
+export default function ChangePasswordForm() {
+    const classes = useStyles();
+    const [passwordVisible, setPasswordVisible] = useState(false);
+    const [
+        passwordConfirmationVisible,
+        setPasswordConfirmationVisible,
+    ] = useState(false);
+
+    const validationSchema = Yup.object().shape({
+        email: Yup.string()
```

Comment:
You can delete `email` and `confirmEmail` as you are not using those in this form